assets/shaders/maptexture.frag.glsl

```diff
@@ -1,13 +1,10 @@
-#version 120
-// total basic standard texture drawing fragment shader
+#version 330 core
 
-// the texture data
-uniform sampler2D texture;
+uniform sampler2D texture_;
```

Comment:
Better rename it `tex` than using underscores because that aligns with other shader source code definitions.

```suggestion
uniform sampler2D tex;
```